src/Builder/ConfigurationEnvironmentBuilder/ArrayConfigurationEnvironmentBuilder.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\Builder\ConfigurationEnvironmentBuilder;
+
+use SprykerSdk\Integrator\IntegratorConfig;
+
+class ArrayConfigurationEnvironmentBuilder implements ConfigurationEnvironmentBuilderInterface
```

Comment:
I am not sure that `*Builder` is an appropriate prefix, e.g *EnvironmentConfigurationValueFormatterStrategy` or similar.